Makefile

```
@@ -50,14 +50,24 @@ pip-compile:
 	pip-compile requirements/dev.in
 	pip-compile requirements/test.in
 
-#########
-# Build #
-#########
+##########
+# Docker #
+##########
+
+# Docker targets are provided for convenience only and are not required in a standard development environment
+
+# Note that the current working directory is mounted under
+# /home/notebook-user/local/ when the image is started with
+# docker-start-api
 
-## docker-build:            builds the docker container for detectron2
 .PHONY: docker-build
 docker-build:
-	PIP_VERSION=${PIP_VERSION}  ./scripts/docker-build.sh
+	PIP_VERSION=${PIP_VERSION} ./scripts/docker-build.sh
+
+.PHONY: docker-start-api
+docker-start-api:
+	docker run -p 8000:8000 --mount type=bind,source=$(realpath .),target=/home/notebook-user/local -t --rm --entrypoint uvicorn unstructured-inference-dev:latest ${PACKAGE_NAME}.api:app --host 0.0.0.0 --port 8000
```

```suggestion
	docker run -p 8000:8000 --mount type=bind,source=$(realpath .),target=/home/notebook-user/local -t --rm unstructured-inference-dev:latest --entrypoint uvicorn ${PACKAGE_NAME}.api:app --host 0.0.0.0 --port 8000
```
...although see my other note about changing the tag from `ml-inference`.

---

actually, when using --entrypoint it needs to come before the image name. re-tested to make sure.
```
$ make docker-start-api
docker run -p 8000:8000 --mount type=bind,source=/home/ec2-user/repos/unstructured-inference,target=/home/\
notebook-user/local -t --rm --entrypoint uvicorn unstructured-inference-dev:latest unstructured_inference.\
api:app --host 0.0.0.0 --port 8000
INFO:     Started server process [1]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
INFO:     Uvicorn running on http://0.0.0.0:8000 (Press CTRL+C to quit)
```

---

(and need to use entrypoint since the Dockerfile defines it)